src/main/java/gitflow/Gitflow.java

```diff
@@ -109,4 +110,5 @@ GitCommandResult trackBugfix(@NotNull GitRepository repository,
                                   @NotNull GitRemote remote,
                                   @Nullable GitLineHandlerListener... listeners);
 
+    GitCommandResult version(@NotNull Project project, GitLineHandlerListener... listeners);
```

Comment:
@OpherV I've removed the @Nullable annotation on the second parameter. Passing null as a varargs is not a good thing IMO.